core-blocks/embed/index.js

```diff
@@ -255,7 +255,7 @@ function getEmbedBlockSettings( { title, description, icon, category = 'embed',
 				return (
 					<Fragment>
 						{ controls }
-						<figure className={ classnames( className, { 'is-video': 'video' === type } ) }>
+						<figure className={ classnames( className, 'wp-block-embed__figure', { 'is-video': 'video' === type } ) }>
```

Comment:
I have no clue as to how to name classes in Gutenberg–is there a doc that points me toward what's right?

Reply:
@aduth pointed me to two links that might help:

- https://github.com/Automattic/eslint-plugin-wpcalypso/blob/master/lib/rules/jsx-classname-namespace.js
- https://github.com/WordPress/gutenberg/blob/master/docs/reference/coding-guidelines.md#css